primitives/header-chain/src/justification.rs

```diff
@@ -315,76 +356,99 @@ where
 			authorities_set_id,
 			&mut signature_buffer,
 		) {
-			return Err(Error::InvalidAuthoritySignature)
+			callbacks.on_invalid_authority_signature(precommit_idx)?;
```

Comment:
If we are treating such justifications as normal now, then maybe let's a test for that?

Reply:
Done.